.github/workflows/build.yml

```diff
@@ -150,6 +150,32 @@ jobs:
           path: |
             */target/*-0-SNAPSHOT.jar
 
+  sonar:
+    name: Sonar Scan
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Set up JDK
+        uses: actions/setup-java@v2
+        with:
+          java-version: 11
+          distribution: 'adopt'
+          cache: 'maven'
+      - name: Cache SonarCloud packages
+        uses: actions/cache@v1
```

Comment:
@nvoxland Any reason we can't use v2?
https://github.com/actions/cache

Reply:
Pushed v2 to see if it works.